12.nadeo-importer/03.how-to-make-a-specularmap-diffusemap-for-objects/docs.md

```diff
@@ -15,8 +16,27 @@ I will explain very quickly what are the specifications and how to save them in
 
 ## What you need
 
-* a graphic manipulation software (ps, gimp etc.)
+* a graphic manipulation software (Adobe Photoshop (PS), Gimp etc.)
+Free & Open Source: https://www.gimp.org/
+Free Trial: https://www.adobe.com/products/photoshop.html
+
 * a dds converter which allows you to set the specific output format (like the nvidia dds tools)
+For the Gimp .dds plugin search google.
+PS Plugin: https://developer.nvidia.com/nvidia-texture-tools-adobe-photoshop
+
+
+## Compare files with originals
+
+Sometimes it is useful to see what nadeo used, so you need the original files.
+They are stored as .zip file in C:\ProgramData\ManiaPlanet\PacksCache
+You need TMValley_HD.zip or TMValley.zip and the Updated files e.g. for Valley,Stadium,Canyon...
+Unzip them at the destination of your choice.
+
+!Info! You have to download the Title in TM before the files are stored there.
```

Comment:
Strange `!Info!` block that doesn't really mean anything in formatting. You might wanna replace it with something more formatted, like this: (I also slightly rewrote the line itself)
```
**Info:** You have to download the titlepack in ManiaPlanet before the files are stored there.
```

Reply:
I dont like wikipedia, github etc. i think this does everything more complicated than needed.
There is no user friendly formatting, handling etc. i really did my best and used the formatting the guy created the page used before. Thank you for editing it.

Reply:
If you need to learn Markdown, [this website](https://www.markdownguide.org/) is a great place to start.